src/main/java/ch/njol/skript/sections/EffSecSendResourcePack.java

```diff
@@ -0,0 +1,214 @@
+package ch.njol.skript.sections;
+
+import ch.njol.skript.Skript;
+import ch.njol.skript.config.SectionNode;
+import ch.njol.skript.doc.Description;
+import ch.njol.skript.doc.Examples;
+import ch.njol.skript.doc.Name;
+import ch.njol.skript.doc.Since;
+import ch.njol.skript.lang.SkriptParser.ParseResult;
+import ch.njol.skript.lang.EffectSection;
+import ch.njol.skript.lang.Expression;
+import ch.njol.skript.lang.Trigger;
+import ch.njol.skript.lang.TriggerItem;
+import ch.njol.skript.variables.Variables;
+import ch.njol.util.Kleenean;
+import ch.njol.util.StringUtils;
+import org.bukkit.event.HandlerList;
+import org.jetbrains.annotations.NotNull;
+import org.jetbrains.annotations.Nullable;
+import org.bukkit.entity.Player;
+import org.bukkit.event.Event;
+import ch.njol.skript.util.Utils;
+import org.jetbrains.annotations.UnknownNullability;
+
+import java.util.List;
+import java.util.UUID;
+import java.util.concurrent.atomic.AtomicBoolean;
+
+@Name("Send Resource Pack")
+@Description({
+	"Request that the player's client download and switch resource packs. The client will download",
+	"the resource pack in the background, and will automatically switch to it once the download is complete.",
+	"The URL must be a direct download link.",
+	"",
+	"The [**resource pack request action**](events.html#resource_pack_request_action) can be used to check",
+	"the status of the sent resource pack request.",
+	"",
+	"Take a look at [**Resource Pack Values**](https://docs.skriptlang.org/effects.html#EffResourcePackValues) to see additional options."
```

Comment:
```suggestion
	"The [**resource pack request action**](events.html#resource_pack_request_action) can be used to check " +
	"the status of the sent resource pack request.",
	"",
	"Take a look at [**Resource Pack Values**](https://docs.skriptlang.org/effects.html#EffResourcePackValues) to see additional options."
```

Also, use HTML tags not markdown.